.github/workflows/build_scoped_rector_php70.yaml

```diff
@@ -34,17 +33,20 @@ jobs:
                     COMPOSER_TOKEN: ${{ secrets.ACCESS_TOKEN }}
 
             # install only prod dependencies - do not use ramsey, it uses cache including "dev", we want to avoid it here
-            -   run: composer install --no-dev --ansi
+            -   run: |
+                    composer install --no-dev --ansi
+                    composer remove phpstan/phpstan
+                    composer require phpstan/phpstan-src
```

Comment:
@TomasVotruba to handle dependency conflict, I think we can do this, remove `phpstan/phpstan` and require `phpstan/phpstan-src`, with `phpstan` dependencies then removed from composer.json target https://github.com/rectorphp/rector/pull/6235/commits/c884f78ea066a56f0a4f86b272885edae30c57da

Reply:
The `phpstan/phpstan-src` is not in the packagist, seems need to be added to "repositories" config:

```json
    "repositories" : [
        {
            "type" : "vcs",
            "url" : "https://github.com/phpstan/phpstan-src",
            "no-api": true
        }
    ]
```

Reply:
it seems not installable:

```bash
  [RuntimeException]                                                                                           
  Failed to clone 
https://github.com/phpstan/phpstan-src
 via https, ssh protocols, aborting.                   
  - 
https://github.com/phpstan/phpstan-src

    Cloning into bare repository '/home/runner/.cache/composer/vcs/https---github.com-phpstan-phpstan-src'...  
    BUG: refs/files-backend.c:2953: initial ref transaction called with existing refs                          
    Aborted (core dumped)                                                                                      
```

Reply:
I used to install it like this, just without "no-api"

Reply:
I usually use `no-api` used to allow non-owner to use, eg: in fork or PR, without no-api, it got the following error:

```bash
Error: Could not authenticate against github.com

  [Composer\Downloader\TransportException]   
  Could not authenticate against github.com  
```

Reply:
@TomasVotruba without no-api, it got error:

```bash
Error: Could not authenticate against github.com

  [Composer\Downloader\TransportException]   
  Could not authenticate against github.com  
```

in https://github.com/rectorphp/rector/pull/6235/commits/df7769606a40667c69e36d4bc1d365377a93cdb1 .

Reply:
This works for me locally:

```bash
{
	"require": {
		"phpstan/phpstan-src": "dev-master"
	},
	"repositories": [
		{
			"type": "vcs",
			"url": "https://github.com/phpstan/phpstan-src"
		}
	]
}
```

The dev packages needs to be completed too.

![image](https://user-images.githubusercontent.com/924196/115984835-5b9cf080-a5a9-11eb-9314-7bdeb6eedd85.png)

Reply:
To modify `composer.json` in command line, you can use this https://getcomposer.org/doc/03-cli.md#modifying-repositories

Reply:
@TomasVotruba that seems because of you are the repo owner so the authentication github always works.

Reply:
undo change root composer.json